lib/cc/engine/duplication.rb

```diff
@@ -35,6 +38,7 @@ def any_enabled_languages?
 
       def languages
         engine_config['languages'] || ENABLED_BY_DEFAULT
+        ['python']
```

Comment:
I'm confused...what's the use of the previous line if we're just returning this array?